azure-templates.yaml

```diff
@@ -0,0 +1,24 @@
+trigger:
+  branches:
+    include:
+      - main
```

Comment:
for this repo the main branch for jsm is `jsm-master`. 
In this scenario we have two approach, rename this branch to main os change the azure main reference to `jsm-master`